models/izhikevich.cpp

```diff
@@ -78,9 +78,9 @@ nest::izhikevich::Parameters_::Parameters_()
 }
 
 nest::izhikevich::State_::State_()
-  : v_( -65.0 ) // membrane potential
-  , u_( 0.0 )   // membrane recovery variable
-  , I_( 0.0 )   // input current
+  : v_( -70.0 )       // membrane potential
```

Comment:
The model implementation in the [original Izhikevich publication](https://www.izhikevich.org/publications/spikes.pdf) uses V_m_init = -65.0, U_m_init = b * V_m_init. Is there particular reason why we don't following it?